superset-frontend/src/dashboard/components/menu/PopoverDropdown.jsx

```diff
@@ -78,6 +78,7 @@ const MenuItem = styled(Menu.Item)`
 class PopoverDropdown extends React.PureComponent {
```

Comment:
Maybe we should add a Storybook entry? Could be a separate PR, or we can do a whole audit of missing storybook entries after we close the books on some of the current audit/transition efforts.

Reply:
I agree with the suggestion in a separate PR. I added this component to our tests audit spreadsheet with the following notes: 

> We should move this to the components folder, change it to a functional component with typescript, create tests and storybook